mpariente/asteroid
# Add Sinkhorn PIT loss

### asteroid/engine/system.py 
- A change was made to pass the epoch number to the PIT wrapper.

### asteroid/losses/pit_wrapper.py
- Add Sinkhorn iteration-based permutation search.
    - Some parameters are still hard coded.
- Possibility of stopping on ASSERT during the validation, which is painful if it occurs in a very heavy training. We may need to devise a way to separate valid and test by flags. Replacing it with the Hungarian algorithm is another option.

### tests
- Check if Sinkhorn and Hungarian return almost the same results for 2 <= N <=9.
- Shape check.
- Check if Sinkhorn-based reordering is correct.
